alacritty/src/display/content.rs

```diff
@@ -350,11 +350,13 @@ impl RenderableCell {
     /// using the named input color, rather than checking the RGB of the background after its color
     /// is computed.
     #[inline]
-    fn compute_bg_alpha(bg: Color) -> f32 {
+    fn compute_bg_alpha(config: &UiConfig, bg: Color) -> f32 {
         if bg == Color::Named(NamedColor::Background) {
             0.
-        } else {
+        } else if config.colors.transparent_background_colors {
             1.
```

Comment:
This is wrong, isn't it? Looks like you just changed the name from `opaque` to `transparent` but you didn't invert the behavior.